app/src/main/java/com/isc/hermes/generators/PointGenerator.java

```diff
@@ -1,37 +1,34 @@
 package com.isc.hermes.generators;
 
+import com.isc.hermes.model.incidents.GeometryType;
+
+import java.util.ArrayList;
 import java.util.List;
 
 /**
  * This method has the responsibility to generate point coordinates data.
  */
-public class PointGenerator extends CoordinateGen {
+public class PointGenerator extends CoordinateGen implements IGeometryGenerator {
 
-    private int maxAttempts;
+    private final int MAXATTEMPTS = 500;
```

Comment:
The conventions for declaring a constant in Java are:
accessModifier static final dataType WORD_WORD_WORD.
In this case it would be private static final int MAX_ATTEMPTS = 500;